server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -184,4 +209,37 @@ public void stop()
     );
   }
 
+  @Provides
+  @Singleton
+  public JettyMonitor getJettyMonitor(Properties props)
+  {
+    return new JettyMonitor(props);
+  }
+
+  public static class JettyMonitor extends AbstractMonitor
+  {
+    private final Map<String, String[]> dimensions;
+
+    public JettyMonitor(Properties props)
+    {
+      this.dimensions = MonitorsConfig.extractDimensions(
+          props,
+          Lists.newArrayList(
+              DruidMetrics.DATASOURCE,
+              DruidMetrics.TASK_ID
+          )
+      );
+    }
+
+    @Override
+    public boolean doMonitor(ServiceEmitter emitter)
+    {
+      final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder();
+      MonitorUtils.addDimensionsToBuilder(
+          builder, dimensions
+      );
+      emitter.emit(builder.build("jetty/numConns", activeConnections.get()));
```

Comment:
not all jetty requests are "query" requests. not sure if it belongs there, renamed it to jetty/numOpenConnections though.